src/components/Stepper/index.tsx

```diff
@@ -163,6 +163,9 @@ type StepperProps = {
   value?: number
 } & Omit<InputHTMLAttributes<HTMLInputElement>, 'size'>
 
+/**
```

Comment:
Need to be specified in the storybook too i thnk ?

Reply:
@chambo-e https://github.com/scaleway/scaleway-ui/pull/1482 is still in review, in meaning time I said we can use this until the PR is merged